source/extensions/filters/common/expr/context.h

```diff
@@ -71,6 +71,7 @@ constexpr absl::string_view URISanLocalCertificate = "uri_san_local_certificate"
 constexpr absl::string_view URISanPeerCertificate = "uri_san_peer_certificate";
 constexpr absl::string_view DNSSanLocalCertificate = "dns_san_local_certificate";
 constexpr absl::string_view DNSSanPeerCertificate = "dns_san_peer_certificate";
+constexpr absl::string_view SHA256PeerCertificateDigest = "sha256_peer_certificate_digest";
```

Comment:
Should this be added to `WrapperFieldValues` per comment above?

Reply:
Yes, thank you for reminding me (I left that comment)!